csrankings-j.csv

```diff
@@ -2436,7 +2436,7 @@ Junzhou Huang,University of Texas at Arlington,http://ranger.uta.edu/~huang,X7Kr
 Junzhou Zhao,Xi'an Jiaotong University,https://junzhouzhao.github.io,hBLT754AAAAJ
 Juraj Hromkovic,ETH Zurich,https://www.inf.ethz.ch/department/faculty-profs/person-detail.html?persid=102124,NOSCHOLARPAGE
 Juraj Somorovsky,Paderborn University,https://www.uni-paderborn.de/person/83504,aPYih6YAAAAJ
-Jürgen Beyerer,Karlsruhe Institute of Technology,http://ies.anthropomatik.kit.edu/mitarbeiter.php?person=beyerer,-BCKzg0AAAAJ
+Jürgen Beyerer,Karlsruhe Institute of Technology,http://ies.iar.kit.edu/mitarbeiter.php?person=beyerer,-BCKzg0AAAAJ

```

Comment:
Web page does not appear to be correct - perhaps https://ies.iar.kit.edu/index.php?